### PR TITLE
ENH: Conda pack

### LINKED ISSUE #117: Use conda-pack for deployment
This will be faster and cleaner than letting conda do its thing. We can take the packed environments and upload them on the github release page here, or onto drive if they are too big. It will also make it much easier to transition to local deployments on the hutch control machines in the future.

### PR DESCRIPTION
**Pull request by ZLLentz**
closes #117 

I will continue to release environments on psbuild the old way, but include this `conda-pack` step and upload the tarballs it creates. This will make it easier to reproduce the exact environment on linux hosts, especially those without internet connections.

### CODE REVIEW

**Comment by klauer** [comment]
I'll take a look but have never used conda pack, outside peering over @hhslepicka 's shoulder. Requesting his review here.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ A simple repo to hold our current and past conda env.yaml files. Each yaml file
...
 - For latest: `source py36env.sh`
-- For specific: `source py36env.sh pcds-0.1.0`
+- For a specific release, set this environment variable first: `export PCDS_CONDA_VER=2.1.0`
+
+To install a pre-built environment on a linux host without conda:
+- Navigate to the release pages on this repo
+- Download the environment tarball (not the source code)
+- `tar -xzf my_env.tar.gz -C my_env`
+- `source my_env/bin/activate`
+- `conda-unpack` (first time setup)
```
Comment (hhslepicka) [inline README.md:15]: Should probably add that after the `conda-unpack` it is advisable to deactivate and re-activate the environment to ensure that scripts are loaded properly.

Comment (ZLLentz) [reply README.md:15]: Thanks, I'll add this!

**Comment by ZLLentz** [comment]
Yes, the tarballs will be uploaded alongside the release notes.

**Comment by ZLLentz** [comment]
Merging while ignoring travis because I know the environment is broken

**Pull request merged**